map/framework.cpp

```diff
@@ -3496,13 +3496,59 @@ void VisualizeFeatureInRect(m2::RectD const & rect, FeatureType & ft, df::DrapeA
 }
 }  // namespace
 
+void VisualizeMwmBorder(df::DrapeApi & drapeApi, std::string const & mwmName)
```

Comment:
Can we have a mode that either chooses the .poly file or unpacks the actual polygon from packed_polygons.bin?